ext4/gpt/types.go

```diff
@@ -0,0 +1,103 @@
+package gpt
+
+import (
+	"encoding/binary"
+	"math"
+
+	"github.com/Microsoft/go-winio/pkg/guid"
+)
+
+// See uefi specification at https://uefi.org/specifications for details
+
+var (
+	SizeOfPMBRInBytes    = binary.Size(ProtectiveMBR{})
+	SizeOfHeaderInBytes  = binary.Size(Header{})
+	SizeOfPartitionEntry = binary.Size(PartitionEntry{})
+
+	ProtectiveMBRStartingCHS       = [3]byte{0x00, 0x02, 0x00}
+	ProtectiveMBREndingCHSMaxArray = [3]byte{0xff, 0xff, 0xff}
+	ProtectiveMBRSizeInLBAMaxValue = math.MaxUint32
+
+	LinuxFilesystemDataGUID = guid.GUID{
+		Data1: 0x0FC63DAF,
+		Data2: 0x8483,
+		Data3: 0x4772,
+		Data4: [8]uint8{0x8E, 0x79, 0x3D, 0x69, 0xD8, 0x47, 0x7D, 0xE4},
+	}
+)
+
+const (
+	BlockSizeLogical                         = 512 // logical block size in bytes
```

Comment:
Can we count on LBA size always being 512 bytes? Should this be a parameter instead?